apps/api/src/scraper/WebScraper/single_url.ts

```diff
@@ -109,6 +109,38 @@ function getScrapingFallbackOrder(
   return scrapersInOrder as (typeof baseScrapers)[number][];
 }
 
+function extractLinks(html: string, baseUrl: string): string[] {
```

Comment:
@calebpeffer could you move this function a extractLinks file inside api/src/scraper/WebScraper/utils?

Reply:
@calebpeffer add docs too lol